uiowa/uiowa
# [Bug] Editing circle list links in WYSIWYG

Resolves https://github.com/uiowa/uiowa/issues/7796
Resolves https://github.com/uiowa/uiowa/issues/7802

# How to test

- Review https://github.com/uiowa/uids/pull/933

```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /node/2411/edit
```

Confirm that you can edit the link in WYSIWYG
Save page and confirm circle list renders as expected
